packages/material-ui/src/FormControlLabel/FormControlLabel.js

```diff
@@ -62,12 +62,12 @@ const FormControlLabel = React.forwardRef(function FormControlLabel(props, ref)
     inputRef,
     label,
     labelPlacement = 'end',
-    muiFormControl,
     name,
     onChange,
     value,
     ...other
   } = props;
+  const muiFormControl = useFormControl();
```

Comment:
`muiFormControl` or `formControl`?

Reply:
We always used `muiFormControl` so I just wanted to keep the diff small.